Sources/Core/AWSSDKHTTPAuth/AWSSigV4Signer.swift

```diff
@@ -5,18 +5,26 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+import Smithy
+import SmithyIdentityAPI
+import SmithyHTTPAPI
+import SmithyHTTPAuthAPI
+import SmithyChecksumsAPI
 import AwsCommonRuntimeKit
 import ClientRuntime
 import Foundation
 
-public class AWSSigV4Signer: ClientRuntime.Signer {
+public class AWSSigV4Signer: SmithyHTTPAuthAPI.Signer {
```

Comment:
This is the regular signer for SigV4.  It is split from `AWSClientRuntime` to the new `AWSSDKHTTPAuth` module.